GitUI/CommandsDialogs/FormRemotes.cs

```diff
@@ -274,7 +274,7 @@ private void InitialiseTabRemotes(string preselectRemote = null)
 
         private void InitialiseTabDefaultPullBehaviors(string preselectLocal = null)
         {
-            var heads = Module.GetRefs(false, true);
+            var heads = Module.GetRefs(false, true).OrderBy(r => r.LocalName).ToList();
```

Comment:
I'm not sure that there is an added value to have branches sorted in a form like this one which is a configuration form (not a form that is part of a daily workflow)...

Reply:
FYI, I just added the sort because there was no sort at all and even if the column are set has sortable, they are not due to the type of list binded 😢